index.html

```diff
@@ -4,10 +4,176 @@
+                  <li class="tags">javaScript</li>
+               </ul>
+                <div class="action-button">
+                   <button class="see-project-button">See Project</button>
+                </div>
+           </div>
+        </div>
+    </main>
+    <section id="about-myself">
+      <div class="primary-info">
+        <h2 class="header2">About
+         <br>
+         Myself
+        </h2>
+        <p id="primary-info-paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h3 class="lets-connect">LET'S CONNECT</h3>
+        <ul class="media-links">
+            <li>
+               <a href="#">
+                  <img src="Images/smm-icons/github-Icon.svg">
+               </a>
+            </li>
+            <li>
+               <a href="#">
+                  <img src="Images/smm-icons/Linkedin-Icon.svg">
+               </a>
+            </li>
+            <li>
+               <a href="#">
+                  <img src="Images/smm-icons/victory-Icon.svg">
+               </a>
+            </li>
+            <li>
+               <a href="#">
+                  <img src="Images/smm-icons/twitter-Icon.svg">
+               </a>
+            </li>
+            <li>
+               <a href="#">
+                  <img src="Images/smm-icons/M-Icon.svg">
+               </a>
+            </li>
+        </ul>
+        <div class="action-button">
+         <button class="see-project-button">Get my resume</button>
+        </div>
+      </div>
+      <div id="tech-info-data">
+         <ul class="tech-info">
+            <li class="tech-info-font">
+               <h3>Languages</h3>
+               <i class="fa fa-angle-up"></i>
+            </li>
+               <ul class="languages-list">
+                  <li class="dev-language">
+                     <img class="image-lang" src="Images/Ellipse JS.png" alt="javaScript">
+                     <span class="lang-span">JavaScript</span>
+                  </li>
+                  <li class="dev-language">
+                     <img class="image-lang" src="Images/Ellipse HTML.png" alt="HTML">
+                     <span class="lang-span">HTML</span>
+                  </li>
+                  <li class="dev-language">
+                     <img class="image-lang" src="Images/Ellipse CSS.png" alt="CSS">
+                     <span class="lang-span">CSS</span>
+                  </li>
+               </ul>
+            <li class="tech-info-font">
+               <h3>Frameworks</h3>
+               <i class="fa fa-angle-right"></i>
+            </li>
+            <hr class="horizontal-line">
+            <li class="tech-info-font">
+               <h3>Skills</h3>
+               <i class="fa fa-angle-right"></i>
+            </li>
+            <hr class="horizontal-line">
+         </ul>
+      </div>
```

Comment:
- [ ] Also the about section, You may add a `margin` and `paddings` at the bottom to stick to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=t6rOGNfHIv0gVnpn-0) as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) and to make your design more beautiful 😃.

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![about](https://user-images.githubusercontent.com/57408419/221093283-c311166f-b683-4ee8-8652-1fde16258564.jpg)| |![about01](https://user-images.githubusercontent.com/57408419/221093360-d0625329-a29c-4e96-9417-5f922bca7ff2.jpg) | |

- [ ] Kindly, make sure you added some `border-radius`  on the top of your about section to stick to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=t6rOGNfHIv0gVnpn-0) as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.)

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![section01](https://user-images.githubusercontent.com/57408419/221094431-48f27d34-d292-4d8a-9651-dda3e14d0fff.jpg)| |![section](https://user-images.githubusercontent.com/57408419/221094555-5c681c05-30c1-4775-91cb-163f4f2353f5.jpg)| |